src/BinOp.chpl

```diff
@@ -1148,7 +1174,12 @@ module BinOp
             var divideBy = makeDistArray(la.size, bigint);
             divideBy = 1:bigint;
```

Comment:
@pierce314159 we're seeing `bigint_bitwise_binops.py` timeout during nightly testing for `>>`. I think the problem is probably here with assigning all elements to a bigint `1` that lives on locale 0. I _think_ you could get rid of the tmp array and just do:

```chpl
forall t in tmp with (var dB = (1:bigint) << val, var local_max_size = max_size) {
```

Reply:
Whoops, I meant this as a comment for the binopVS case. For the binopVV cases you should be able to do something like:

```chpl
forall (t, ri) in zip(tmp, ra) with (var local_max_size = max_size) {
  var dB = 1:bigint;
  dB <<= ri;
  t /= dB;
```

Reply:
Opened https://github.com/Bears-R-Us/arkouda/pull/2178